tensornetwork/contractors/opt_einsum_paths/optimal_path.py

```diff
@@ -0,0 +1,29 @@
+import opt_einsum
```

Comment:
I think this file and the `optimal_path_test` file got accidentally readded during a merge. Can you remove these?